content/Hardware/Generic/How-to-do-a-loopback-test.md

```diff
@@ -34,8 +38,30 @@ A failed loopback test indicates a problem with one or more of the following:
    * **Arduino IDE:** *Tools > Serial Monitor*
    * **Web Editor:** Select *Monitor* in the sidebar.
 
-8. Enter a message and click Send (the baud rate does not matter). Your message should immediately be echoed by the board, and appear in the output field below.
+8. Enter a message and click Send. Your message should immediately be echoed by the board, and appear in the output field below.
 
    ![Performing a loopback test in the Arduino IDE serial monitor.](img/serial_monitor_loopback_test.gif "Performing a loopback test in the Arduino IDE serial monitor")
 
-  *Performing a loopback test in the Arduino IDE serial monitor.*
+   *Performing a loopback test in the Arduino IDE serial monitor.*
+
+## If the loopback test failed
+
+A failed loopback test failed indicates that the USB to TTL serial adapter is damaged and that the board should be replaced. [Contact us](https://www.arduino.cc/en/contact-us/) for questions about warranty, or visit [the Arduino Store](https://store.arduino.cc/).
+
+> **Note:** This test will always fail for the CH340 USB to serial chip used on some derivate boards.
```

Comment:
Should we add the link to How to spot counterfeit board article here?

Reply:
I would say no, as the relevant difference is using a different chip, regardless of whether the derivate would be considered a counterfeit.